google_guest_agent/network/manager/netplan_linux.go

```diff
@@ -144,7 +144,12 @@ func (n netplan) Name() string {
 
 // Configure gives the opportunity for the Service implementation to adjust its configuration
 // based on the Guest Agent configuration.
-func (n netplan) Configure(ctx context.Context, config *cfg.Sections) {
+func (n *netplan) Configure(ctx context.Context, config *cfg.Sections) {
```

Comment:
the key aspect should be: netplan will not merge all configs across possible config dirs and then picking the higher priority (like systemd-networkd and everything else does), we want to install our dropins to `/run/netplan` - when it's debian 12 (we can investigate if we can assume it as the default to all systems).

Reply:
Updated to do so. Netplan documentation says we can rely on it, unsure if distros are disabling it in some cases for some reason.

Reply:
We can rely on this.